x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/swimlane/swimlane.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="43" height="43" viewBox="0 0 43 43" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We are currently working on the reducing the bundle size for triggersActionUi plugin and replacing all SVG files with lazy loaded react components([PR](https://github.com/elastic/kibana/pull/97770) in review), which it supported by EUI IconType. It would be nice to have this changes for a Swimlane connector in the follow up PR

Reply:
@YulNaumenko Done! [`d64875e` (#100086)](https://github.com/elastic/kibana/pull/100086/commits/d64875e9ab0387d499444e7d8b5101190a3ed02f)